bitcoin/BRWalletManager.c

```diff
@@ -1364,6 +1364,7 @@ BRWalletManagerCreateTransaction (BRWalletManager manager,
     pthread_mutex_unlock (&manager->lock);
 
     if (NULL != txnWithState) {
+        BRTransactionWithStateSetResolved (txnWithState);  // Always resolved if created
```

Comment:
Modifications should be in the lock, I believe. To be honest, the gets should be there too (a mistake on my part).

Reply:
Addressed w/ the 'gets' too.